third_party/rust/Cargo.toml

```diff
@@ -55,8 +55,8 @@ rsa = "0.8"
 rusb = "0.8.1"
 serde = { version="1.0", features=["serde_derive"] }
 serde_derive = "1.0"
-
 serde_bytes = "0.11"
+serde-hjson = "0.9"
```

Comment:
@pamaury have more context on this, but I think that we had issues trying to parse some hjson files in the repo with the rust implementation. So unless we really need the hjson features, here I would suggest using json.

Reply:
@engdoreis is right, but if you _do_ need to use hjson then we found the [deser-hjson](https://crates.io/crates/deser-hjson) crate to have fewer bugs (but not zero) than serde-hjson.

Reply:
Unless we want to serialize hjson, I would indeed suggest that we stick `deser-hjson` which we already use and which is working well (we successfully submitted upstream bug fixes).

Reply:
This is fixed with the updated PR.